NPE in PerconaAddColumnChange when constraint without nullable attribute

To reproduce, try the following changelog:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<databaseChangeLog
        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:liquibasePercona="http://www.liquibase.org/xml/ns/dbchangelog-ext/liquibase-percona"
        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
        http://www.liquibase.org/xml/ns/dbchangelog-ext/liquibase-percona https://raw.githubusercontent.com/liquibase/liquibase-percona/main/src/main/resources/dbchangelog-ext-liquibase-percona.xsd">

    <changeSet id="1" author="alice">
        <createTable tableName="person">
            <column name="created" type="datetime"/>
        </createTable>
    </changeSet>
    <changeSet  id="2" author="bob">
        <addColumn tableName="person">
            <column name="address" type="varchar(255)">
                <constraints unique="true"/>
            </column>
        </addColumn>
    </changeSet>
</databaseChangeLog>
```

`<constraints unique="true"/>` kills liquibase-percona:

```
SEVERE [liquibase.integration] Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "liquibase.change.ConstraintsConfig.isNullable()" is null
liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "liquibase.change.ConstraintsConfig.isNullable()" is null
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "liquibase.change.ConstraintsConfig.isNullable()" is null
        at liquibase.ext.percona.PerconaAddColumnChange.convertColumnToSql(PerconaAddColumnChange.java:83)
        at liquibase.ext.percona.PerconaAddColumnChange.generateAlterStatement(PerconaAddColumnChange.java:74)
        at liquibase.ext.percona.PerconaChangeUtil.generateStatements(PerconaChangeUtil.java:101)
        at liquibase.ext.percona.PerconaAddColumnChange.generateStatements(PerconaAddColumnChange.java:61)
        at liquibase.change.AbstractChange.generateStatementsVolatile(AbstractChange.java:341)
        at liquibase.change.AbstractChange.warn(AbstractChange.java:412)
        at liquibase.changelog.visitor.ValidatingVisitor.visit(ValidatingVisitor.java:133)
        at liquibase.changelog.ChangeLogIterator$2.lambda$run$0(ChangeLogIterator.java:128)
```

The changelog is correct. If I set `-Dliquibase.percona.defaultOn=false`, here is the output:
```
INFO [liquibase.ui] Running Changeset: db.changelog.xml::2::bob
INFO [liquibase.changelog] Columns address(varchar(255)) added to person
INFO [liquibase.changelog] ChangeSet db.changelog.xml::2::bob ran successfully in 24ms
```

Environment:
- Liquibase version 4.25.0
- liquibase-percona 4.25.0
- OpenJDK 17.0.9

The bug is very old. I found it trying to use liquibase-percona 4.15.0. You should expect `null` in any field of ConstraintsConfig. See liquibase.change.core.CreateTableChange – they always do a null-check first.

--- a/src/main/java/liquibase/ext/percona/PerconaAddColumnChange.java
+++ b/src/main/java/liquibase/ext/percona/PerconaAddColumnChange.java
@@ -80,7 +80,7 @@ public class PerconaAddColumnChange extends AddColumnChange implements PerconaCh
     String convertColumnToSql(AddColumnConfig column, Database database) {
         String nullable = "";
         ConstraintsConfig constraintsConfig = column.getConstraints();
-        if (constraintsConfig != null && !constraintsConfig.isNullable()) {
+        if (constraintsConfig != null && constraintsConfig.isNullable() != null && !constraintsConfig.isNullable()) {
             nullable = " NOT NULL";
         } else {
             nullable = " NULL";

--- a/src/test/java/liquibase/ext/percona/PerconaAddColumnChangeTest.java
+++ b/src/test/java/liquibase/ext/percona/PerconaAddColumnChangeTest.java
@@ -317,4 +317,22 @@ public class PerconaAddColumnChangeTest extends AbstractPerconaChangeTest<Percon
                 + "ADD CONSTRAINT fk_test_column FOREIGN KEY (testColumn) REFERENCES _person_new(id)",
                 c.convertColumnToSql(column, database));
     }
+
+    @Test
+    void testConvertColumnWithoutExplicitNullable() {
+        PerconaAddColumnChange change = getChange();
+        Database database = getDatabase();
+
+        AddColumnConfig column = new AddColumnConfig();
+        column.setName("address");
+        column.setType("varchar(255)");
+
+        ConstraintsConfig constraints = new ConstraintsConfig();
+        constraints.setNullable((Boolean) null); // make sure, it is null
+        constraints.setUnique(true);
+        column.setConstraints(constraints);
+
+        Assertions.assertEquals("ADD COLUMN address VARCHAR(255) NULL, ADD UNIQUE (address)",
+                change.convertColumnToSql(column, database));
+    }
 }

